recipes/xbpch/meta.yaml

```diff
@@ -0,0 +1,54 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
```

Comment:
Please remove all the boilerplate comments from the example recipes :smile:

Reply:
Haha whoops - thanks for the clarification